### PR TITLE
Updates formValidation and adminHelpers

### PR DESCRIPTION
**Pull request by mino323**
This pr update formValidation and adminHelpers to include module related information. 
This will allow future the admin module page components to use the form components. 
Admin module page is the next step in https://github.com/garageScript/c0d3-app/issues/1192
FormValidations will now have module schema to error check module information. 
adminHelpers now changes 'lessonId' in a form, from a string  to a int which is require for a graphql mutation.

adminHelper test were updated to include 'lessonId'

### CODE REVIEW

File: helpers/formValidation.tsx [hunk 1]
```diff
@@ -37,6 +37,15 @@ const lessonSchema = Yup.object({
   chatUrl: Yup.string()
 })
 
+const moduleSchema = Yup.object({
```
Comment (flacial) [inline helpers/formValidation.tsx:40]: `moduleSchemaValidation` to match the file naming pattern of `name+vValidation`?

File: helpers/admin/adminHelpers.ts [hunk 2]
```diff
@@ -58,6 +58,10 @@ export const makeGraphqlVariable = (options: any, addProp?: any) => {
     res.id = parseInt(res.id ? res.id + '' : '')
   }
 
+  if (res.hasOwnProperty('lessonId')) {
```
Comment (songz) [inline helpers/admin/adminHelpers.ts:61]: Question: How does `challenges` work? DOesn't `challenge` also have a field for `lessonId`?

**Review by flacial — APPROVED** [review]

File: helpers/admin/adminHelpers.ts (hunk 2 above)
Comment (mino323) [reply helpers/admin/adminHelpers.ts:61]: It doesn't ask for the lesson. When you click a lesson all the challenges currently come with it. When you add a challenge you are on the lesson page already.

File: helpers/formValidation.tsx (hunk 1 above)
Comment (songz) [reply helpers/formValidation.tsx:40]: yeah good idea to keep things consistent.

**Review by SahilKalra98 — APPROVED** [review]

**Pull request closed without merging**